packages/nextjs/cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
```

Comment:
I was getting this in terminal : 

![Screenshot 2024-07-31 at 2 14 41 PM](https://github.com/user-attachments/assets/276493a4-0c57-40d0-91d7-b637c09a7079)

Doing `"baseUrl": ".",` seems to solve it as mentioned here https://github.com/cypress-io/cypress/issues/15724#issuecomment-2193921083